apps/backoffice-v2/src/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/hooks/useCaseGenerationForm/utils/prepare-context.ts

```diff
@@ -0,0 +1,15 @@
+import { generateEntityId } from '@/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/hooks/useCaseGenerationForm/utils/generate-entity-id';
+import { AnyObject } from '@ballerine/ui';
+
+export const prepareContext = (formData: AnyObject): AnyObject => {
```

Comment:
We already use `create`, `compose`, `generate`, and `build`. Let's stick to ideally around one. 🙏🏼 
```suggestion
export const createInitialContext = (formData: AnyObject): AnyObject => {
```